src/recommendations/RecommendationsPage.jsx

```diff
@@ -21,10 +21,29 @@ const RecommendationsPage = (props) => {
 
   const { formatMessage } = useIntl();
   const [selectedRecommendationsType, setSelectedRecommendationsType] = useState(RECOMMENDATIONS_OPTION_LIST[1]);
-  const [recommendations] = useState([]);
+  const [recommendations, setRecommendations] = useState([]);
+  const [popularRecommendations, setPopularRecommendations] = useState([]);
+  const [trendingRecommendations, setTrendingRecommendations] = useState([]);
+
+  useEffect(() => {
```

Comment:
We don't need this `useEffect`, `popularRecommendations` and `trendingRecommendations` state. Just use the `recommendations` state in the second `useEffect` as you are already doing and inside it should be like this:

```
 const popularProducts = JSON.parse(getConfig().POPULAR_PRODUCTS);
 const trendingProducts = JSON.parse(getConfig().TRENDING_PRODUCTS);
 setRecommendations(option.value === 'trending' ? trendingRecommendations : popularRecommendations);
```

Reply:
Addressed.